packages/desktop-client/src/components/budget/report/ReportComponents.tsx

```diff
@@ -88,13 +89,29 @@ export function IncomeHeaderMonth() {
 }
 
 type GroupMonthProps = {
+  month: string;
   group: { id: string; is_income: boolean };
 };
-export const GroupMonth = memo(function GroupMonth({ group }: GroupMonthProps) {
+export const GroupMonth = memo(function GroupMonth({
+  month,
+  group,
+}: GroupMonthProps) {
   const { id } = group;
 
   return (
-    <View style={{ flex: 1, flexDirection: 'row' }}>
+    <View
+      style={{
+        flex: 1,
+        flexDirection: 'row',
+        ...(monthUtils.isCurrentMonth(month)
+          ? {
+              backgroundColor: theme.budgetHeaderCurrentMonth,
+            }
+          : {
+              backgroundColor: theme.budgetHeaderOtherMonth,
+            }),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏would you mind changing this line & the others too (I won't add a comment for all the occurrences since there are quite a lot)
```suggestion
          backgroundColor: monthUtils.isCurrentMonth(month) ? theme.budgetHeaderCurrentMonth : theme.budgetHeaderOtherMonth,
```

Reply:
A lot more places still need this patch applied.

Reply:
Should be all fixed now.